presto-docs/src/main/sphinx/connector/hive-caching.rst

```diff
@@ -177,16 +177,16 @@ The implementation of the cache exposes a `number of metrics
 :doc:`inspect these and other metrics directly in Presto with the JMX connector
 or in external tools </admin/jmx>`.
 
-The following example query returns the total amount of data fetched by the
-BookKeeper on the coordinator and all the separate nodes on the workers in the
-default async mode:
+If you are using JMX connector, basic caching statistics are exposed via
```

Comment:
Remove "If you are using JXM connector" sins this is already said earlier.

Basic caching statistics are available in the ...

Reply:
Should it be 
```
The table ``jmx.current."rubix:catalog=<catalog_name>,name=stats`` 
```
Or 
```
The ``jmx.current."rubix:catalog=<catalog_name>,name=stats`` table
```

Latter sound more proper to me, but I may be very wrong here :)

Reply:
either is fine